### PR TITLE
New endpoint to query the Pulp2-Pulp3 mapping

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
https://pulp.plan.io/issues/5634
closes #5634

### CODE REVIEW

File: pulp_2to3_migration/app/viewsets.py [hunk 1]
```diff
@@ -94,3 +95,26 @@ class Pulp2ContentViewSet(NamedModelViewSet, mixins.RetrieveModelMixin, mixins.L
...
+        fields = {
+            'pulp2_repo_id': ['exact', 'in'],
+        }
+
+
+class Pulp2to3MappingViewSet(NamedModelViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin):
+    """
+    ViewSet for Pulp2to3Mapping model.
+    """
+    endpoint_name = 'pulp2to3mapping'
```
Comment (dkliban) [inline pulp_2to3_migration/app/viewsets.py]: Let's change this endpoint name to 'pulp2repositories'.

Maybe also change the class names for the viewset and serializer?

Comment (fabricio-aguiar) [reply pulp_2to3_migration/app/viewsets.py]: @goosemania wdyt?

Comment (ipanova) [reply pulp_2to3_migration/app/viewsets.py]: +1

**Comment by ipanova** [comment]
@fabricio-aguiar can you add the new endpoint to the user docs? https://github.com/pulp/pulp-2to3-migration/blob/master/README.md#user-guide

**Pull request merged**